addons/hr_expense/static/src/mixins/document_upload.js

```diff
@@ -96,6 +99,10 @@ export const ExpenseDocumentUpload = (T) => class ExpenseDocumentUpload extends
         this.fileInput.el.click();
     }
 
+    increateUploadsProcessing() {
+        this.uploadsProcessing++;
```

Comment:
just `this.uploadsProcessing++;` in useBus(this.env.bus, "change_file_input" ... is enough I think